configs/application/config.json

```diff
@@ -61,7 +51,18 @@
 		"showNativeTabs": true,
 		"floatingTitlebarComponent": "Floating Titlebar"
 	},
-	"betaFeatures": {},
+	"betaFeatures": {
```

Comment:
This is the deprecated location of these props, while it may have been necessary to make it work with whatever version of Finsemble you linked with, you might consider not committing this.